### ISSUE TITLE
screen reader interferes  with keyboard shortcuts when typing

### ISSUE THREAD

**Issue by smoke-j** [darmentrout/ableplayer-for-wp#3]
When using a screen reader you cannot type on a page that also has able player on it. This is a big issue with something like comments on wordpress. A screen reader has to go into focuss mode, or forms mode for typing into an edit box. This is also the mode which is used in all screen readers to pass keyboard commands along to html audio and video players on a webpage. When in forms or focus mode your keys are being typed into an edit box but also sent to able player. So every time you hit the space bar between words in your typed message it is pausing and unpausing the media in able player constantly. I ahve not tested with all other shortcuts, mainly because just the space bar bug is enough to make people go a bit crazy. This is an awesome player, just this one bug that has been happing for a while now that is forcing me to not use it as much. A simple way to disable space bar to play and pause would be really appreciated.
